client/data/site-verticals/types.ts

```diff
@@ -0,0 +1,20 @@
+export interface SiteVerticalsVerticalApi {
+	id: string;
+	title: string;
+}
+
+export interface SiteVerticalsVertical {
```

Comment:
Hmm, this shape (`{ value, label, category }`) looks too tied to an `<option/>` inside a `<select/>`, so I'm not sure if this is best placed inside a `client/data` package.

What if:
- we put this in the select component directly instead, and
- we return the actual shape from `useSiteVerticalsQuery()`

So, something like this:

```
export interface VerticalResponse { // 'SiteVerticalsVerticalApi' sounds a bit weird, what do you think? :)
  id: string;
  title: string;
  ... etc
}

export interface Vertical
  id: string;
  parentId: string;
  name: string;
  ... etc
}
```

Another reason for this is when in the future there are other use cases which requires the actual shape of a vertical, it won't break the select component.

What do you think?

Reply:
Yeah, I agree with using `VerticalResponse` and putting this in the select component directly.

Reply:
Updated